plugins/search/src/components/SearchBar/SearchBar.tsx

```diff
@@ -40,6 +40,11 @@ export const SearchBarBase = ({
   placeholder: overridePlaceholder,
 }: PresenterProps) => {
   const configApi = useApi(configApiRef);
+  const inputRef = useRef<HTMLInputElement>();
+
+  useEffect(() => {
+    inputRef?.current?.focus();
```

Comment:
Do we want to auto-focus the input field every time this component is rendered, or do we want this to be configurable based on the context it's rendered in (e.g. via a prop or something)?

Reply:
To me it made sense to do it every time it renders just because in 9 times out of 10 you enter a page/modal and the first thing you want to do is to search. I couldn't come up with any situations when you don't want the input field to be focused, but let me know if you can!

Reply:
I'm thinking of scenarios where search may be a secondary action on the page (e.g. imagine a version of the TechDocs Home that included a search bar, but also other relevant content about how to create/use TechDocs within an organization).

You're probably right though, 90% of the time, it should probably auto-focus.

Reply:
good point, let me change it so that its possible to pass a prop, but that it defaults to focused. Does that sounds good?

Reply:
@iamEAP  Decided to add the focused prop, but not default to focused as its more clear to provide the prop when you want the search bar to be focused than the other way around. So check my changes (also added a storybook example)